src/celeritas/random/XorwowRngEngine.hh

```diff
@@ -51,15 +67,46 @@ class XorwowRngEngine
     //! Highest value potentially generated
     static CELER_CONSTEXPR_FUNCTION result_type max() { return 0xffffffffu; }
 
-    // Construct from state
-    inline CELER_FUNCTION
-    XorwowRngEngine(StateRef const& state, TrackSlotId const& id);
+    // Construct from state and persistent data
+    inline CELER_FUNCTION XorwowRngEngine(ParamsRef const& params,
+                                          StateRef const& state,
+                                          TrackSlotId tid);
+
+    // Initialize state
+    inline CELER_FUNCTION XorwowRngEngine& operator=(Initializer_t const&);
 
     // Generate a 32-bit pseudorandom number
     inline CELER_FUNCTION result_type operator()();
 
+    // Jump ahead \c num_skip steps
+    inline CELER_FUNCTION void skipahead(ull_int num_skip);
+
+    // Jump ahead \c num_skip subsequences (\c num_skip * 2^67 steps)
+    inline CELER_FUNCTION void skipahead_subsequence(ull_int num_skip);
```

Comment:
It would be good to annotate that this function is public only for testing purposes (or maybe it would be better to just make it private?)

Reply:
Made them private.